custom_components/waste_collection_schedule/waste_collection_schedule/source/sunderland_gov_uk.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+
+import requests
+import re
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Sunderland City Council"
+DESCRIPTION = "Source for sunderland.gov.uk services for Sunderland City Council, UK."
+URL = "https://www.sunderland.gov.uk/"
+API_URL = "https://webapps.sunderland.gov.uk/WEBAPPS/WSS/Sunderland_Portal/Forms/bindaychecker.aspx"
+HEADERS = {"user-agent": "Mozilla/5.0"}
+TEST_CASES = {
+    "Test_001": {"postcode": "SR4 7PU", "address": "191 Cleveland Road"},
+    "Test_002": {"postcode": "SR3 2DW", "address": "43 Hill Street"},
+    "Test_003": {"postcode": "SR4 8RJ", "address": "17 Sutherland Drive"},
+}
+ICON_MAP = {"Recycle": "mdi:recycle", "House": "mdi:trash-can", "Garden": "mdi:leaf"}
+
+
+class Source:
+    def __init__(self, postcode: str, address: str):
+        self._postcode = str(postcode).replace(" ", "+")
+        self._address = str(address)
+        self._ddlAddress: str = None
+
+    def get_viewstate(self, content: str) -> dict:
+        tags = {}
+        soup = BeautifulSoup(content, "html.parser")
+        hidden_tags = soup.findAll("input", type="hidden")
+        for _, tag in enumerate(hidden_tags):
```

Comment:
This is just useless code as `enumerate` adds the index, and you just assign it to `_`, so `for tag in hidden_tags:` does exactly the same thing.

Reply:
done